docs/user/docker.md

```diff
@@ -30,91 +45,100 @@ To save the experiment data generated in the container, you need to specify a
-container is based on with the date and time the container was launched.
+cloned repository. The data is saved in a folder named `data` inside the current
+working directory. You can override that by using the variable `DATA_PATH` and
+passing the *absolute path* to the output directory.
+
+```
+make run-headless RUN_CMD="..." DATA_PATH="/home/xyz/my_garage_exp"
+```
+
+By default, docker generates random names for containers. If you want to specify
+a name for the container, do so with the variable `CONTAINER_NAME`.
 
-If you want to specify another name for the container, do so with the variable
-`CONTAINER_NAME`:
 ```
-make run-headless RUN_CMD="..." CONTAINER_NAME="my_container"
+make run-headless RUN_CMD="..." CONTAINER_NAME="my_container_123"
 ```
 
 If you need to use MuJoCo, you need to place your key at `~/.mujoco/mjkey.txt`
 or specify the corresponding path through the MJKEY_PATH variable:
+
 ```
 make run-headless RUN_CMD="..." MJKEY_PATH="/home/user/mjkey.txt"
 ```
 
-If you require to pass addtional arguments to the the make commands, you can
-use the variable ADD_ARGS, for example:
+If you require to pass additional arguments to docker build and run commands,
+you can use the variables BUILD_ARGS and RUN_ARGS, for example:
+
 ```
-make build-headless ADD_ARGS="--build-arg MY_VAR=123"
-make run-headless ADD_ARGS="-e MY_VAR=123"
+make build-headless BUILD_ARGS="--build-arg MY_VAR=123"
+make run-headless RUN_ARGS="-e MY_VAR=123"
 ```
 
-#### Prerequisites
+## Nvidia image
 
-Be aware of the following prerequisites to build the image.
+The garage nvidia images come with CUDA 10.2.
 
-- Install [Docker CE](https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce). Tested
-  on version 18.09.0.
-- Install [Docker Compose](https://docs.docker.com/compose/install/#install-compose). Tested
-  on version 1.23.2.
+### Prerequisites for nvidia image
 
-Tested on Ubuntu 16.04. It's recommended to use the versions indicated above
-for docker-ce and docker-compose.
+Additional to the prerequisites for the headless image, make sure to have:
 
-## nvidia image
+- Install the latest NVIDIA driver, tested
+  on [nvidia-390](https://tecadmin.net/install-latest-nvidia-drivers-ubuntu/)
+- [Install nvidia-docker2](https://github.com/NVIDIA/nvidia-docker#ubuntu-160418042004-debian-jessiestretchbuster)
+
+Tested on Ubuntu 18.04 & 20.04.
+
+### Run a pre-compiled garage-nvidia image
 
 The same commands for the headless image mentioned above apply for the nvidia
 image, except that the image name is defined by `rlworkgroup/garage-nvidia`.
+
 For example, to execute a launcher file:
+
 ```
 docker run -it --rm rlworkgroup/garage-nvidia python examples/tf/trpo_cartpole.py
 ```
 
-### Build and run the image
+### Build and run the nvidia image
 
 The same rules for the headless image apply here, except that the target names
 are the following:
+
 ```
 make build-nvidia
 make run-nvidia
```

Comment:
doesn't run-nvidia automatically run `build-nvidia`?

Reply:
It does. So should we not put both build-headless and build-nvidia in here?

Reply:
yeah -- so just let `make` do the work.